src/plugins/formulas/__tests__/initialization.spec.js

```diff
@@ -0,0 +1,696 @@
+import HyperFormula, {
+  FunctionPlugin
+} from 'hyperformula';
+import plPL from 'hyperformula/es/i18n/languages/plPL';
+
+describe('Formulas general', () => {
+  const debug = false;
+  const id = 'testContainer';
+
+  beforeEach(function() {
+    this.$container = $(`<div id="${id}"></div>`).appendTo('body');
+    this.$container2 = $(`<div id="${id}-2"></div>`).appendTo('body');
+  });
+
+  afterEach(function() {
+    if (debug) {
+      return;
+    }
+
+    if (this.$container) {
+      try {
+        if (this.$container.handsontable('getInstance')) {
+          destroy();
+        }
+      } catch (e) {
+        // In some of the test cases we're manually destroying the Handsontable instances, so 'getInstance' may
+        // throw a post-mortem error.
+        if (!e.message.includes('instance has been destroyed')) {
+          throw e;
+        }
+      }
+
+      this.$container.remove();
+    }
+
+    if (this.$container2) {
+      try {
+        if (this.$container2.handsontable('getInstance')) {
+          this.$container2.handsontable('getInstance').destroy();
+        }
+      } catch (e) {
+        // In some of the test cases we're manually destroying the Handsontable instances, so 'getInstance' may
+        // throw a post-mortem error.
+        if (!e.message.includes('instance has been destroyed')) {
+          throw e;
+        }
+      }
+      this.$container2.remove();
+    }
+  });
+
+  describe('Single Handsontable setup', () => {
+    it('should throw an error, when no `hyperformula` key was passed to the `formulas` settings', () => {
+      /* eslint-disable no-console */
+      const errorSpy = jasmine.createSpyObj('error', ['test']);
+      const prevError = console.error;
+
+      console.error = (...args) => {
+        errorSpy.test(...args);
+        prevError(...args);
+      };
+
+      const hot = handsontable({
+        data: getDataSimpleExampleFormulas(),
+        formulas: true,
+        licenseKey: 'non-commercial-and-evaluation'
+      });
+
+      expect(errorSpy.test).toHaveBeenCalledWith(
```

Comment:
Because it doesn't actually throw anything -> https://github.com/handsontable/handsontable/pull/7823#discussion_r618265756. Not the prettiest way to test it, I know.